notes/2020-10-02.md

```diff
@@ -30,52 +30,58 @@ ca_dds_df
 ```{code-cell} ipython3
 print(ca_dds_df)
 ```
-
+we can do df.index to look at what the index is in our dataframe
 ```{code-cell} ipython3
 ca_dds_df.index
 ```
-
+columns are tuple values. We can index to the first column as below. As we can see by running this code, this column has three names initially. 
 ```{code-cell} ipython3
 ca_dds_df.columns[0]
 ```
-
+By the following code we can assign the first column as the index of original dataframe. If we assign "False" to the "inplace", it will make the changes in a new dataframe.
```

Comment:
```suggestion
We can use this to set the first column as the index of original DataFrame, without having to copy that whole long name manually. The default of `set_index` method is to modify a copy of the DataFrame and return that because `inplace = False`. We want to change our existing DataFrame so we use `inplace=True`
```

The backticks will render as code in the built website.